src/components/Header/Header.css

```
@@ -0,0 +1,23 @@
+.Linq {
+    text-align: center;
+    padding: 10px 0;
+    height: 40px;
+    box-shadow: var(--shadow);
+}
```

```suggestion
.Linq {
    display: flex;
    justify-content: space-around;
    text-align: center;
    padding: 10px 0;
    height: 40px;
    box-shadow: var(--shadow);
}
```

Add the justification so that the header links space properly across mobile and desktop